src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java

```diff
@@ -400,9 +400,11 @@ public final class TranslationConstants
     @NonNls
     public static final String BUILDER_BUILDING_TOO_LOW                                             = "entity.builder.messagebuildtoolow";
     @NonNls
-    public static final String COLONIST_DIED                                                        = "block.blockhuttownhall.messagecolonistdead";
+    public static final String COLONIST_FULL                                                        = "block.blockhuttownhall.messagecolonistfull";
     @NonNls
```

Comment:
why full?

Reply:
Because I feel like it better reflects the use of the changed translation (the translation is now only `The colonist XY` rather than the full `The colonist XY died of AB at X Y Z`; see the changes to manual_en_us.json).

Reply:
COLONIST_DESC maybe

Reply:
Changed to COLONIST_DESC and WORKER_DESC.